test/python/providers/test_backend_configuration.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
With the suggested change regarding `from_dict`, these tests might be rendered obsolete - instead, can you tune `python.test_schemas.TestSchemaExamples#test_examples_are_valid` so it uses the specific subclasses (should be a matter of updating `obj_map` and explicitly including `backend_configuration_openpulse_example.json`, as it is excluded by the `if` ad the end of the method)?